wheatley/main.py

```diff
@@ -405,6 +414,11 @@ def console_main(override_args: Optional[List[str]], stop_on_join_tower: bool) -
     except InvalidURLError as e:
         sys.exit(f"Bad value for '--url': {e}")
 
+    try:
+        parse_start_row(args.start_row)
```

Comment:
I think that we should fully _parse_ the row here rather than just verifying it, because by only passing around strings (which are not inherently valid) we run the risk of accidentally passing an invalid row into `Bot`.

So we'd want something like:
```python
try:
    starting_row: Row = parse_start_row(args.start_row)
except:
   ...
```

In fact, I think it would make sense to move this into `create_row_generator` so that we only have to pass the unparsed args into it.